devops/ci.yml

```diff
@@ -210,36 +166,21 @@ jobs:
       sudo pip install wheel OutputCheck
     displayName: 'Dependencies'
 
-  - script:
-      echo "##vso[task.setvariable variable=PKG_NAME]verona-llvm-x86_64-macos-release-$(LLVMCommit)"
-    displayName: 'Set LLVM Package'
-
-  - script: |
-      set -eo pipefail
-      curl -s --output $(PKG_NAME).tar.gz https://verona.blob.core.windows.net/llvmbuild/$(PKG_NAME)
-    displayName: 'Download package'
-
-  - bash: |
-      set -eo pipefail
-      ./utils/llvm/setup-llvm-builddir.sh $(PKG_NAME).tar.gz
-    displayName: 'Setup LLVM Build dir'
-    workingDirectory: '$(Build.SourcesDirectory)'
-
   - task: CMake@1
     displayName: 'CMake'
     inputs:
       cmakeArgs: |
-        .. -DCMAKE_BUILD_TYPE=$(BuildType) -DENABLE_ASSERTS=ON -DVERONA_CI_BUILD=On -DSNMALLOC_CI_BUILD=On -DRT_TESTS=$(RTTests)
+        .. -DCMAKE_BUILD_TYPE=$(BuildType) -DENABLE_ASSERTS=ON -DVERONA_CI_BUILD=On -DRT_TESTS=$(RTTests)
 
   - script: |
       set -eo pipefail
-      make -j 4 install
+      make -j 4
```

Comment:
I know it was there already, but hard-coding `4` here isn't helpful. There's a way to get the number of cores on Mac.

Reply:
TODO in #198